modules/chronograf-server/main.tf

```diff
@@ -0,0 +1,59 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CREATE AN EC2 INSTANCE TO RUN CHRONOGRAF
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_instance" "chronograf_server" {
```

Comment:
- Would you ever run chronograf in a cluster for high availability or scalability? 
- If not, should we run it in an ASG of size 1 for auto recovery?
- Should we offer the ability to attach a Route 53 domain name to the server rather than using IPs that could change?
- Or, alternatively, would it make sense to put a load balancer in front of it with a Route 53 domain name + TLS cert?

Reply:
+1 on the single node ASG

Reply:
> * Would you ever run chronograf in a cluster for high availability or scalability?

Yup!

> * If not, should we run it in an ASG of size 1 for auto recovery?

Agreed!

> 
    * Should we offer the ability to attach a Route 53 domain name to the server rather than using IPs that could change?
    * Or, alternatively, would it make sense to put a load balancer in front of it with a Route 53 domain name + TLS cert?

I'm inclined to go with the load balancer route. Since we already have modules for that